nextflow_schema.json

```diff
@@ -193,6 +193,16 @@
                     "type": "string",
                     "default": "${projectDir}/assets/references.bibtex",
                     "description": "Reference bibtex file for the report, should not be changed."
+                },
```

Comment:
why is it a variable if it shouldn't be changed?

Reply:
There was apparently some nextflow change; now it seems the files in assets/ cannot be accessed by processes unless they are provided to the respective module as channels.